lib/ssl/src/ssl_connection.hrl

```diff
@@ -67,7 +67,7 @@
                         early_data_accepted = false :: boolean(), %% TLS 1.3
                         allow_renegotiate = true                    ::boolean(),
                         %% Ext handling
-                        hello,                %%:: #client_hello{} | #server_hello{}            
+                        handshake_continue = hello,  %% hello | client_hello_versions (when entering user_hello in TLS-1.3) | continue (after user_hello)
```

Comment:
I think the name if this new field `handshake_continue` is confusing. Maybe it would be better if it was named `handshake_mode` or `handshake_progress` and since it is a field in the record `handshake_env` the prefix `handshake` can be omitted and we land in `mode` or `progress`. What do you think?

Reply:
Another possible name could be `continue_status`

Reply:
Sure